app/controllers/repos_controller.rb

```diff
@@ -116,7 +138,7 @@ def show_releases
       format.html do
         @result_types = fetch_categories
       end
-
+      format.json { render json: generate_json }
```

Comment:
If `generate_json` works with only with `@charts`, it would be more clear to pass that variable to the method (`generate_json(charts)`) and no longer rely on instance variable.